integration_tests/CMakeLists.txt

```diff
@@ -400,6 +400,7 @@ RUN(NAME logical_binop1      LABELS cpython llvm)
 RUN(NAME vec_01              LABELS cpython llvm c)
 RUN(NAME test_str_comparison LABELS cpython llvm c)
 RUN(NAME test_bit_length     LABELS cpython llvm c)
+RUN(NAME test_bit_count      LABELS cpython llvm c)
```

Comment:
@diivi the labels here enable that particular test for the specified backend. Here, you enabled the `test_bit_count` test case for `llvm` as well as `c`. The tests passed for `llvm`, but were failing for `c` as `visit_IntegerBitCount` was not implemented in the `c` backend.

Reply:
Ah, okay thanks! I referred the int.bit_length() method so kept it the same as that. Could you help me understand the difference/benefit of adding the visit_IntegerBitCount function in the c backend too? And why this wasn't done for visit_IntegerBitLength? Thanks!

Reply:
>And why this wasn't done for visit_IntegerBitLength?

I think `visit_IntegerBitLength()` is already supported in the `c` backend. It is defined here:
https://github.com/lcompilers/lpython/blob/ba525e69ad328d28a492134b1d824a9e44a898d0/src/libasr/codegen/asr_to_c_cpp.h#L1337-L1348

>Could you help me understand the difference/benefit of adding the visit_IntegerBitCount function in the c backend too?

The more backends support a feature the better for us. (Ideally, every backend would/should support every feature). In your PR, since you implemented most of the required portion and the CI tests were failing for `c` backend for the test case `test_bit_count` (due to missing `visit_IntegerBitCount`), we went ahead and added support for it so that the CI passes. 

What we could have also done is to not enable the `c` backend for that specific test (since, the `c` backend didn't have support for it). And mayber later (in subsequent PRs) we could have added support for that test case in the `c` backend.